test/integration/default/serverspec/windows_spec.rb

```diff
@@ -2,25 +2,29 @@
 require 'spec_helper'
 
 expected_rules = [
-  %r{firewall add rule name="prepend" description="prepend" dir=in service=any protocol=tcp localip=any localport=7788 interfacetype=any remoteip=any remoteport=any action=allow},
```

Comment:
Hey @martinb3, if the rule is not bound to any service, meaning any service can use this rule, service=any should not be coded. If service=any is coded, then it is only allowed for a service with name "any".